### PR TITLE
add a postgres config file to try and make pg14 faster

### PR DESCRIPTION
**Pull request by jcrides**
Adding a postgresql.conf file with settings to try and make PG14 run tests faster.

### CODE REVIEW

**Comment by noisybit** [comment]
Hard to parse the active settings vs commented out, can please you summarize the changes?

**Comment by jcrides** [comment]
> Hard to parse the active settings vs commented out, can please you summarize the changes?

This is just updating the settings to match what are in the PG11 container we had.

**Comment by noisybit** [comment]
Did you test it? Are configs forward compatible?

**Comment by jcrides** [comment]
> Did you test it? Are configs forward compatible?

Yes I tested it. This starts with the config file from PG14 then I enabled all the same options as our PG11 container.

**Review by noisybit — APPROVED** [review]
Random note, I think this repo is open to the world? I was able to view this PR when I wasn't signed in.
